spec/controllers/posts_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
```

Comment:
- [ ] Following the project requirements, your project should Have `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory.